libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -650,7 +650,7 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 			{
 				// Check if zero (out of stack or not enough balance).
 				m_context << Instruction::ISZERO;
-				m_context.appendConditionalInvalid();
+				m_context.appendConditionalRevert();
```

Comment:
Yes, similar, but also this depends on other factors like timing and so on. Happy to discuss all these!